integration-tests/modules/__tests__/inventory/index.spec.ts

```diff
@@ -284,6 +284,97 @@ medusaIntegrationTestRunner({
         })
       })
 
+      describe("Update inventory levels", () => {
+        let locationId
+        let inventoryItemId
+        beforeEach(async () => {
+          const invItemReps = await api.post(
+            `/admin/inventory-items`,
+            { sku: "test-sku" },
+            adminHeaders
+          )
+
+          inventoryItemId = invItemReps.data.inventory_item.id
+
+          const sl = await appContainer
```

Comment:
todo: as per our convention (old one) we shouldn't use one or two letter variable name, only in inline function

Reply:
I know I can just read the resolve to understand that it is the stock location module and therefore it is probably a stock location, but better getting use to it now to stop doing that :)